ksql-functional-tests/src/test/java/io/confluent/ksql/test/TopologyFileGenerator.java

```diff
@@ -86,10 +86,6 @@
     private TopologyFileGenerator() {
     }
 
-    public static void main(final String[] args) throws Exception {
```

Comment:
Would you mind adding a link in the JavaDoc of this class to the test people should use to run this, i.e. link to the new `ManualTopologyFileGeneratorTest` test. Otherwise, people may struggle to find it.